src/main/resources/user/walkthrough/Filtering.md

```diff
@@ -107,15 +107,17 @@ EQUAL will only return Entities and Edges with identifiers that match the seed e
 - if you provide an Edge seed, you will only get back Edges that have the same source, destination and directed values.
 
 RELATED will return the EQUAL results (as above) and additional Entities and Edges:
-- if you provide an Entity seed, you will also get back Edges that have have the same vertex value as source or destination.
-- if you provide an Edge seed, you will also get back Entities that have have the same source or destination vertices.
+- if you provide an Entity seed, you will also get back Edges that have have the same source or destination as the vertex value.
+- if you provide an Edge seed, you will also get back Entities that have have the same vertex value as the source or destination.
 
-As the seedMatching flag has now been deprecated, to run equivalent Operations, there are some examples below.  As the default for 
-seedMatching is RELATED, if that is currently used nothing will need to change.
-There is one limitation however, if you have a seedMatching = EQUAL and specify both Edges and Entities that will have to now be
-done under 2 Operations in a chain as there can only be one View applied globally to all input.
+#### Deprecation
+As the seedMatching flag has now been deprecated, to run equivalent Operations with EQUAL seedMatching, you must instead use a View.
+As the default for seedMatching is RELATED, if you use this then nothing will change.
+If you instead set seedMatching to EQUAL, then you can refer to the examples below on how to replace with a View:
 
-OLD:
+For edges:
+
+OLD (SeedMatching):
```

Comment:
It might be useful to add an underline, bold or heading to 'For edges:' so it's clear this is at a higher level.